repose-aggregator/core/repose-core-api/src/test/scala/org/openrepose/core/filter/AbstractConfiguredFilterTest.scala

```diff
@@ -143,6 +167,18 @@ class AbstractConfiguredFilterTest
 
       filter.getInitialized shouldBe true
     }
+
+    it("should call doConfigurationUpdated") {
```

Comment:
so in the case of the configuration updated method when it is called is super important, because the end of that method is what marks the filter as ready. We have no test making sure that some one doesn't accidentally go and move the call to the new method to after that and potentially fubar up the filter running.